gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -26,6 +26,116 @@ syntax = "proto2";
 option java_package = "com.google.transit.realtime";
 package transit_realtime;
 
+message StopTime{
+  required int64 stop_sequence = 1;
+  required uint64 arrival_time = 2;
+  required uint64 departure_time = 3;
+  required string stop_id = 4;
+  optional TranslatedString stop_headsign = 5;
+  optional PickupType pickup_type = 6;
+  optional DropOffType drop_off_type = 7;
+  optional float shape_dist_traveled = 8;
+
+  enum PickupType{
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+  enum DropOffType{
+    REGULAR_DROP_OFF = 0;
+    NO_DROP_OFF = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+}
+
+message StopTimeProperties{
+  optional string platform_id = 1;
+  optional TranslatedString stop_headsign = 2;
+  optional PickupType pickup_type = 3;
+  optional DropOffType drop_off_type = 4;
+  optional float shape_dist_traveled = 5;
+  optional string stop_id = 6;
+
+  enum PickupType{
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+  enum DropOffType{
+    REGULAR_DROP_OFF = 0;
+    NO_DROP_OFF = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+}
+
+message VehicleProperties{
+  optional WheelchairAccessibleStatus wheelchair_accessible = 1;
```

Comment:
For clarity, all enums should be labeled with a default value:

`optional WheelchairAccessibleStatus wheelchair_accessible = 1 [default = UNKNOWN_WHEELCHAIR_ACCESSIBILITY];`